flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcTableSourceSinkFactoryTest.java

```diff
@@ -269,6 +269,17 @@ public void testJdbcValidation() {
 			fail("exception expected");
 		} catch (ValidationException ignored) {
 		}
+
+		// connection.max-retry-timeout property is smaller than 1 second
+		try {
+			Map<String, String> properties = getBasicProperties();
+			properties.put("connector.connection.max-retry-timeout", "100ms");
+
+			TableFactoryService.find(StreamTableSourceFactory.class, properties)
+				.createStreamTableSource(properties);
+			fail("exception expected");
+		} catch (ValidationException ignored) {
+		}
```

Comment:
Add the expected message when catching the exception.

Reply:
other property don't  add the expected message,i think it stand for the expected message  when catching the exception  in the test perspective

Reply:
Hi @xiaoHoly , yes, you are right. But it may be better to have a validation whether the exception information is expected.
If you add that, you will find the message is not expected, for exception is not propagated in https://github.com/apache/flink/blob/59a212fa7750286b88486dd6894b32fa2aec3ecb/flink-table/flink-table-common/src/main/java/org/apache/flink/table/descriptors/DescriptorProperties.java#L1658

Reply:
yes，i have noticed  what your said. Therefore, i think catch ValidationException is the current reasonable method